### PR TITLE
Don't mention 'switch' so lightbulb text can apply equally to VB and C#

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/12235

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

**Comment by jmarolf** [comment]
:+1:

**Comment by jasonmalinowski** [comment]
:+1:

File: src/Features/Core/Portable/PopulateSwitch/PopulateSwitchDiagnosticAnalyzer.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ namespace Microsoft.CodeAnalysis.PopulateSwitch
     [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
     internal class PopulateSwitchDiagnosticAnalyzer : DiagnosticAnalyzer, IBuiltInAnalyzer
     {
-        private static readonly LocalizableString s_localizableTitle = new LocalizableResourceString(nameof(FeaturesResources.Add_missing_switch_cases), FeaturesResources.ResourceManager, typeof(FeaturesResources));
+        private static readonly LocalizableString s_localizableTitle = new LocalizableResourceString(nameof(FeaturesResources.Add_missing_cases), FeaturesResources.ResourceManager, typeof(FeaturesResources));
         private static readonly LocalizableString s_localizableMessage = new LocalizableResourceString(nameof(WorkspacesResources.PopulateSwitch), WorkspacesResources.ResourceManager, typeof(WorkspacesResources));
```
Comment (Pilchie) [inline src/Features/Core/Portable/PopulateSwitch/PopulateSwitchDiagnosticAnalyzer.cs]: There is still http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/WorkspacesResources.Designer.cs,d27d0278e7a660a1,references.

Comment (Pilchie) [inline src/Features/Core/Portable/PopulateSwitch/PopulateSwitchDiagnosticAnalyzer.cs]: Ping?

Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/PopulateSwitch/PopulateSwitchDiagnosticAnalyzer.cs]: Good point.  WIll address.

**Pull request merged**